### DISCUSSION TITLE
Audio over USB

### DISCUSSION THREAD

**Discussion by aidasb123** [BimmerGestalt/AAIdrive#496] [Q&A] [answered]
Hi guys,

Would like to know if there is possible to play Audio (Spotify) over USB connection?
Phone is connects with AAI app and car over USB cable. Spotify app shows in the media list. I'm able to control Spotify with car (skip and change tracks), but the sound only comes from the phone and not from the car stereo.
Android 11. 
BMW ID6

Regards,

**Comment by hufman** [comment, marked as answer]
USB Audio is unlikely to work, I believe the support for the car's USB Audio protocol was removed from Android Oreo in 2017. If you were able to find a phone that still ran an older version (and if the firmware had the support compiled in), AAIdrive should automatically detect this and configure the car to play music from the USB connection.
With newer phones, you will need to set your car to play music from the phone's Bluetooth connection, and use AAIdrive as an improved control interface over USB. However, your ID6 should support running the app connection over Bluetooth, which will automatically handle the audio connection properly.
